### PR TITLE
feat(neuron-ui): add import keystore

### PR DESCRIPTION
**Pull request by Keith-CY**
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by ashchan** [comment]
Really nice. Our current new/import screens are aligned vertically, can we style the new screens the same way? @Keith-CY

File: packages/neuron-ui/src/locales/en.json [hunk 1]
```diff
@@ -42,6 +42,8 @@
       "welcome-to-nervos-neuron": "Welcome to Neuron",
       "create-or-import-your-first-wallet": "Create or import your first wallet",
       "create-new-wallet": "Create a Wallet",
+      "import-mnemonic": "Import mnemonic words",
```
Comment (ashchan) [inline packages/neuron-ui/src/locales/en.json]:
```suggestion
      "import-mnemonic": "Import Mnemonic Seed",
```

File: packages/neuron-ui/src/locales/en.json [hunk 2]
```diff
@@ -42,6 +42,8 @@
...
+      "import-keystore": "Import a keystore file",
```
Comment (ashchan) [inline packages/neuron-ui/src/locales/en.json]:
```suggestion
      "import-keystore": "Import from Keystore",
```

File: packages/neuron-ui/src/locales/en.json [hunk 3]
```diff
@@ -59,6 +61,22 @@
...
       "same-password": "The password and confirm password should be same"
     },
+    "import-keystore": {
+      "label": {
+        "path": "Keystore File",
+        "name": "Wallet Name",
+        "password": "Password"
+      },
+      "placeholder": {
+        "path": "Click to select the Keystore",
```
Comment (ashchan) [inline packages/neuron-ui/src/locales/en.json]:
```suggestion
        "path": "Click to select the keystore file",
```

File: packages/neuron-wallet/src/locales/en.ts [hunk 4]
```diff
@@ -20,6 +20,8 @@ export default {
         backup: 'Backup Current Wallet',
         delete: 'Delete Current Wallet',
         'change-password': 'Change Password',
+        'import-mnemonic': 'Import Mnemonic Words',
```
Comment (ashchan) [inline packages/neuron-wallet/src/locales/en.ts]:
```suggestion
        'import-mnemonic': 'Import Mnemonic Seed',
```

File: packages/neuron-wallet/src/locales/en.ts [hunk 5]
```diff
@@ -20,6 +20,8 @@ export default {
...
+        'import-keystore': 'Import Keystore File',
```
Comment (ashchan) [inline packages/neuron-wallet/src/locales/en.ts]:
```suggestion
        'import-keystore': 'Import from Keystore',
```

**Comment by Keith-CY** [comment]
> Really nice. Our current new/import screens are aligned vertically, can we style the new screens the same way? @Keith-CY

Updated in https://github.com/nervosnetwork/neuron/pull/862/commits/f8a479673742a23ac9d7428b9cf32f52b1804267

image

**Pull request merged**